### PR TITLE
[WIP] Plug-in: stay hydrated command

### PR DESCRIPTION
**Pull request by Rehkloos**

### CODE REVIEW

File: plugins/hydrate/hydrate.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  event: false,
+  type: 5004,
+  command: 'hydrate',
+  permissions: [],
+  cooldown: 60,
+  settings: false,
+  credits: `Made by Rehkloos`,
+  execute(client) {
+
+    var THIRTY_MINUTES = 1000 * 60 * 30;
```
Comment (Bioblaze) [inline plugins/hydrate/hydrate.js:16]: Can we change this to TIME_BETWEEN_DRINKS

File: plugins/hydrate/hydrate.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      client.sendMessage('Streamer disabled this command');
+    } else {
+      let startDate = Date().toString();
+      console.log(`starting hydration messages at ${startDate}`);
+      client.sendMessage('Great, I will make sure the streamer stays hydrated! 💧 ');
+      var interval = setInterval(function() {
+        let dateTime = Date().toString();
+        console.log(`sending message at ${dateTime}`);
+        client.sendMessage('Streamer, I recommend you drink 240ml (8 oz) of water! I will remind you again in 30 minutes');
+      }, THIRTY_MINUTES); // wait this many milliseconds before reminding again
```
Comment (Bioblaze) [inline plugins/hydrate/hydrate.js:29]: Can we change this to TIME_BETWEEN_DRINKS

File: plugins/hydrate/hydrate.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      console.log('Please change active to true for command to work');
+      client.sendMessage('Streamer disabled this command');
+    } else {
+      let startDate = Date().toString();
+      console.log(`starting hydration messages at ${startDate}`);
+      client.sendMessage('Great, I will make sure the streamer stays hydrated! 💧 ');
+      var interval = setInterval(function() {
+        let dateTime = Date().toString();
+        console.log(`sending message at ${dateTime}`);
+        client.sendMessage('Streamer, I recommend you drink 240ml (8 oz) of water! I will remind you again in 30 minutes');
```
Comment (Bioblaze) [inline plugins/hydrate/hydrate.js:28]: we should calculate the TIME_BETWEEN_DRINKS using Date, to find out how long it is between the next drink, and 8 oz? every 3mins? wtf? you trying to drown the streamer?

Comment (Rehkloos) [reply plugins/hydrate/hydrate.js:27]: 8 oz is half a bottle of water (16.9 are standard water bottles)

**Pull request merged**